assets/user-manifests-bootstrapper/user-manifests-bootstrapper-pod.yaml

```diff
@@ -83,6 +83,10 @@ spec:
           # Replace the global certs configmap here because it's too large to oc apply
           oc create configmap -n openshift-controller-manager openshift-global-ca --from-file ca-bundle.crt=/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem --dry-run -o yaml > /tmp/openshift-global-ca
           oc replace -n openshift-controller-manager -f /tmp/openshift-global-ca --force
+      resources:
+        requests:
+          memory: "100Mi"
```

Comment:
Given you saw `43Mi` as the highest memory usage, I think `100Mi` may be too much of a buffer.  I think we should use set somewhere between `50Mi` and `75Mi`?

Reply:
Updated to 75